entrypoint.sh

```diff
@@ -41,6 +41,23 @@ cloud-api-adaptor-aws aws \
 	-socket /run/peerpod/hypervisor.sock
 }
 
+azure() {
+set -x
+
+cloud-api-adaptor-azure azure \
+  -subscriptionid "${AZURE_SUBSCRIPTION_ID}" \
+  -clientid "${AZURE_CLIENT_ID}" \
+  -secret "${AZURE_SECRET}" \
+  -tenantid "${AZURE_TENANT_ID}" \
+  -region "${AZURE_REGION}" \
+  -instance-size "${AZURE_INSTANCE_SIZE}" \
+  -resourcegroup "${AZURE_RESOURCE_GROUP}" \
+  -subnetid "/subscriptions/${AZURE_SUBSCRIPTION_ID}/resourceGroups/${AZURE_RESOURCE_GROUP}/providers/Microsoft.Network/virtualNetworks/${AZURE_VM_NAME}VNET/subnets/${AZURE_VM_NAME}Subnet" \
+  -securitygroupid "/subscriptions/${AZURE_SUBSCRIPTION_ID}/resourceGroups/${AZURE_RESOURCE_GROUP}/providers/Microsoft.Network/networkSecurityGroups/${AZURE_VM_NAME}NSG" \
+  -imageid "/subscriptions/${AZURE_SUBSCRIPTION_ID}/resourceGroups/${AZURE_RESOURCE_GROUP}/providers/Microsoft.Compute/images/${AZURE_IMAGE}" \
+  -cri-runtime-endpoint /run/containerd/containerd.sock
```

Comment:
@bpradipt - I've not really seen much of the Azure work, so apologies if this is a silly question, but given the change in #280, is it worth setting `CRI_RUNTIME_ENDPOINT="/run/containerd/containerd.sock"` in `install/overlays/azure/kustomization.yaml`, then swapping `-cri-runtime-endpoint /run/containerd/containerd.sock` for `${optionals}` here to match the other cloud providers?

Reply:
Good point @stevenhorsman.
@knrt10 can you please make the changes as suggested by @stevenhorsman to align with other providers ?

Reply:
updated. Please review